src/reducers/index.js

```diff
@@ -0,0 +1,19 @@
+import { createStore } from 'redux';
+import rootReducer from './rootReducer';
+
+const initialState = [
+  {
+    id: 1,
+    title: 'DNA',
+    category: 'Genetics',
+  },
+  {
+    id: 2,
+    title: 'RNA',
+    category: 'Biotechnology',
+  },
+];
```

Comment:
### [Optional]
Kindly be aware that you can use uuidv4 to automatically generate ids. read more 👉 [npm uuid](https://www.npmjs.com/package/uuidv4)

Example:
```
import uuid from 'uuidv4'
id: uuid(): {title: 'DNA', category: 'Genetics'}
```